### ISSUE TITLE
ELECTRA: Pre-training Text Encoders as Discriminators Rather Than Generators 

### ISSUE THREAD

**Issue by icoxfog417** [arXivTimes/arXivTimes#1539]
## 一言でいうと

言語モデルの学習で、MASKで隠した単語を当てるのでなく紛らわしい単語に置換した単語を検出する手法の提案。前段に通常のMLM=Masked Language Model(小サイズ)を置き、穴埋めされた=紛らわしい単語に置き換えられた文字列を本体の言語モデルに送って学習する。BERT/RoBERTaを性能/効率で上回る。

![image](https://user-images.githubusercontent.com/544269/71939642-c41a5f80-31f6-11ea-9d56-be32a15b0d62.png)

### 論文リンク

https://openreview.net/forum?id=r1xMH1BtvB

### 著者/所属機関

Kevin Clark, Minh-Thang Luong, Quoc V. Le, Christopher D. Manning

* Stanford

### 投稿日付(yyyy/MM/dd)

2019/9/26

**Comment by icoxfog417** [comment]
2020/3/9時点ではSQuAD 2.0のsingle modelでトップ。

https://rajpurkar.github.io/SQuAD-explorer/

**Comment by icoxfog417** [comment]
実装
https://github.com/google-research/electra

Transformerでも投入されるよう
https://github.com/huggingface/transformers/issues/1425

**Comment by icoxfog417** [comment]
PyTorch実装。論文精度が再現できたとのこと

https://github.com/richarddwang/electra_pytorch
